flexdll.c

```diff
@@ -434,6 +433,7 @@ static void unlink_unit(dlunit *unit) {
 static void *find_symbol_global(void *data, const char *name) {
   void *sym;
   dlunit *unit;
+  (void)data;
```

Comment:
This puzzled me a bit. I suppose this is needed to avoid an unused parameter warning?

Reply:
Yes, exactly.

Reply:
It had puzzled me in the draft review I started a while back too! Please could there be a comment, therefore?

Reply:
Added a comment to that effect.